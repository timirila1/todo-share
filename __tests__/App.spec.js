import 'react-native';
import React from 'react';
import App from '../App';
import renderer from 'react-test-renderer';

it('Render main screen', async () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
});
