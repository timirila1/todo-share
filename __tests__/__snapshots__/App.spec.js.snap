// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render main screen 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    enabled={false}
    hitSlop={
      Object {
        "right": 50,
      }
    }
    maxDeltaY={20}
    minOffsetX={5}
    onGestureEvent={[Function]}
    onHandlerStateChange={[Function]}
  >
    <View
      style={
        Object {
          "backgroundColor": "#eee",
          "flex": 1,
          "flexDirection": "column-reverse",
          "overflow": "hidden",
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <View
          pointerEvents="auto"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "transform": Array [
                Object {
                  "translateX": 0,
                },
                Object {
                  "translateY": 0,
                },
              ],
            }
          }
        >
          <View
            accessibilityElementsHidden={false}
            style={
              Object {
                "backgroundColor": "#fff",
                "flex": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "flexDirection": "column-reverse",
                  "overflow": "hidden",
                }
              }
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    accessible={true}
                    isTVSelectable={true}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "borderRadius": 20,
                        "bottom": 40,
                        "position": "absolute",
                        "right": 40,
                      }
                    }
                  >
                    <View
                      style={
                        Object {
                          "backgroundColor": "#fa6c35",
                          "borderRadius": 20,
                          "paddingBottom": 10,
                          "paddingLeft": 20,
                          "paddingRight": 20,
                          "paddingTop": 10,
                        }
                      }
                    >
                      <Text />
                    </View>
                  </View>
                </View>
              </View>
              <View
                style={
                  Object {
                    "backgroundColor": "white",
                    "paddingLeft": 40,
                    "paddingRight": 20,
                    "paddingTop": 80,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#5e1c73",
                      "fontSize": 48,
                      "fontWeight": "bold",
                    }
                  }
                >
                  Boards
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
